model/historical_test_data.go

```diff
@@ -113,6 +101,31 @@ func (d *HistoricalTestData) Find(ctx context.Context) error {
 	return nil
 }
 
+func (d *HistoricalTestData) getBucket(ctx context.Context) (pail.Bucket, error) {
+	if d.bucket == "" {
+		conf := &CedarConfig{}
```

Comment:
I wonder if we should cache some of this? Maybe worth opening a ticket.

Reply:
I cached the bucket, I don't think theres anything else to cache here. Unless you want to globally cache it like in evergreen?

Reply:
We can't cache it globally because of the cycle. we can do some rearranging if need be to cache it in between requests if it becomes an issue.